app/controllers/samples_controller.rb

```diff
@@ -1258,6 +1259,29 @@ def coverage_viz_data
     }
   end
 
+  # GET /samples/taxon_with_reads_suggestions
+  # Get taxon search suggestions, where taxon are restricted to the provided sample ids.
```

Comment:
nit: "where taxa are restricted"

Reply:
OMG this whole time I thought taxon was the plural of taxa. I've been doing this all wrong.